pulumi/aws/kic-helm-chart/__main__.py

```diff
@@ -27,9 +27,33 @@ def project_name_from_project_dir(dirname: str):
 
 def build_chart_values(repository: dict) -> helm.ChartOpts:
     values: Dict[str, Dict[str, typing.Any]] = {
-        'controller': {
-            'healthStatus': True
-        }
+      'controller': {
+          'healthStatus': True,
+          'nginxplus': False,
```

Comment:
`nginxplus` is defined below, so it doesn't need to be redefined here.

Reply:
Fixed (removed)